### DISCUSSION TITLE
How to write a simple neural network?

### DISCUSSION THREAD

**Discussion by kenhuang1964** [modular/modular#920] [General]
Hey guys, I'm wondering how can I write a simple neural network using Mojo?

**Comment by erfanzar** [comment]
Check https://github.com/erfanzar/EasyDel Rep and go to the mojo beta branch /lib/mojo/EasyDel/matix_ and there you can find many functions and structures for matrixs neural networks and ...

**Reply by erfanzar** [reply]
https://github.com/erfanzar/EasyDeL/blob/mojo-beta/lib/mojo/EasyDel/matrix_/matrix_struct.%F0%9F%94%A5

**Reply by kenhuang1964** [reply]
Awesome! Thank you legend!
